execution/rerun/rerun.go

```diff
@@ -37,32 +37,27 @@ import (
 )
 
 const (
-	dotGauge     = ".gauge"
-	infoFileName = "runInfo.json"
+	DotGauge   = ".gauge"
```

Comment:
There is already a constant for .gauge folder defined in common. We should be using that variable everywhere.
.gauge folder can be used in other packages too. So why export it here?